code/mobility-services/mobility-services.js

```diff
@@ -189,4 +198,44 @@ $(document).on("knack-view-render.any", function (event, page) {
 
     customizeLoginButton(viewId);
   }
-});
\ No newline at end of file
+});
+
+/****************************************/
+/*** Dropdown Menu Buttons Navigation ***/
+/****************************************/
+function dropdownMenuItem(recordId, route, linkName) {
+  return (
+    `<li class="kn-button">\
+      <a href="#application-operating-authority/business-information/${recordId}/${route}/${recordId}">\
+        <span>${linkName}</span>\
+      </a>\
+    </li>`)
+}
+
+// Dictonary of views needing dropdown menu in editable Operating Authority (OA) pages
+var viewNameOA = {687:"1 - Service Type",676:"2 - Business Information",689:"3 - Insurance", 
+691:"4 - Authorized Person", 693:"5 - Vehicle Information", 695:"6 - Review and Submit"};
```

Comment:
it would be nice to run this new code through a formatter. we typically use [prettier](https://prettier.io/), and there are plugins for most of the major IDEs.

for example, it'd be nice to have this object prettified:

```javascript
var viewNameOA = {
  687: "1 - Service Type",
  676: "2 - Business Information",
  689: "3 - Insurance",
  691: "4 - Authorized Person",
  693: "5 - Vehicle Information",
  695: "6 - Review and Submit",
};
```